README.md

```diff
@@ -3,6 +3,7 @@
 [![Build status](https://github.com/neofinancial/mongo-memory-server-action/workflows/ci/badge.svg)](https://github.com/neofinancial/mongo-memory-server-action/actions)
 [![Test status](https://github.com/neofinancial/mongo-memory-server-action/workflows/pull-request/badge.svg)](https://github.com/neofinancial/mongo-memory-server-action/actions)
 [![Publish status](https://github.com/neofinancial/mongo-memory-server-action/workflows/publish/badge.svg)](https://github.com/neofinancial/mongo-memory-server-action/actions)
+[![coverage](https://coverage.neotools.ca/api/badge/master/4EUt9DTZZpn3u1KfBtNpjn)](https://coverage.neotools.ca/coverage/neofinancial/mongo-memory-server-action)
```

Comment:
Not sure where to get the link for the Codecov badge.

Reply:
That link provides the badge :+1: (see [this reference page](https://github.com/dwyl/repo-badges))